pkgpanda/build/__init__.py

```diff
@@ -773,6 +774,10 @@ def cache_abs(filename):
 
     final_buildinfo['sources'] = sources
 
+    if builder.has('sysctl'):
```

Comment:
Actually, this whole chunk can go, we just need the `pkginfo['sysctl'] = builder.take('sysctl')` below.